xonsh/aliases.py

```diff
@@ -282,7 +284,7 @@ def xexec(args, stdin=None):
         except FileNotFoundError as e:
             return 'xonsh: ' + e.args[1] + ': ' + args[0] + '\n'
```

Comment:
oh yes... Maybe we shouldn't add non-zero return codes to this function. It is the equivalent to python exec so it should only use Exceptions. What is you thought on that?